plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/json/NetworkSettings.java

```diff
@@ -17,7 +17,7 @@
 
 /** @author andrew00x */
 public class NetworkSettings {
-    private String   ipAddress;
+    private String   iPAddress;
```

Comment:
Yes, `ipAddress` is conventional name , but `iPAddress` is needed for parsing value from json. Exactly like in `iPPrefixLen`, `iPv6Gateway` in the same class.